liblangutil/Scanner.cpp

```diff
@@ -76,7 +76,7 @@ string to_string(ScannerError _errorCode)
 		case ScannerError::IllegalHexDigit: return "Hexadecimal digit missing or invalid.";
 		case ScannerError::IllegalCommentTerminator: return "Expected multi-line comment-terminator.";
 		case ScannerError::IllegalEscapeSequence: return "Invalid escape sequence.";
-		case ScannerError::IllegalCharacterInString: return "Invalid character in string.";
+		case ScannerError::IllegalCharacterInString: return "Invalid character in string. If you want to use unicode characters, use a unicode\"...\" string literal.";
```

Comment:
OK, let's make it more precise:
```suggestion
		case ScannerError::IllegalCharacterInString: return "Invalid character in string. If you want to use non-ASCII characters, use a unicode\"...\" string literal.";
```